### DISCUSSION TITLE
v3.2.75

### DISCUSSION THREAD

**Discussion by danharrin** [filamentphp/filament#12702] [General]
Apologies, this fixes some broken form component styling that was compiled in v3.2.74.

**Full Changelog**: https://github.com/filamentphp/filament/compare/v3.2.74...v3.2.75

<hr /><em>This discussion was created from the release <a href='https://github.com/filamentphp/filament/releases/tag/v3.2.75'>v3.2.75</a>.</em>

**Comment by fatihdurmaz** [comment]
rich editor bug 3.2.75

![Ekran görüntüsü 2024-05-09 022354](https://github.com/filamentphp/filament/assets/71706970/2f460ef7-2cf1-489c-a2d0-d342e6c3f2fb)

**Comment by chrispage1** [comment]
Same as @fatihdurmaz I can no longer use the rich editor on this version. Rolled back to .73 and all is fine.

**Comment by danharrin** [comment]
.76 fixes the rich editor, Trix has literally no upgrade guidefrom v1-v2 that I can find and it seems to have broken everything.

**Reply by chrispage1** [reply]
Thanks Dan!
